ide/o.n.swing.dirchooser/src/org/netbeans/swing/dirchooser/DirectoryChooserUI.java

```diff
@@ -237,15 +252,18 @@ public String getDialogTitle(JFileChooser fc) {
         fc.getAccessibleContext().setAccessibleDescription(title);
         return title;
     }
-    
+
+    private static boolean useNBShortcutsPanel() {
+        return UIManager.getLookAndFeel().getID().startsWith("FlatLaf") && true/*some prop*/; // TODO just for testing
```

Comment:
Let's look at necessary UI keys rather than adding any more LAF checks in the codebase!

Reply:
I moved a few things around in this pass.